### PR TITLE
feat(components): [cascader-panel] add multiple selectAll(#806)

### LINKED ISSUE #806: Cascader: add `checkAll` option
**Related Components**

Cascader, Casceder Panel

**Description**

Add `checkAll` option and `checkAllText` option, which only useful in `multiple` mode.
Like this:
image
And sure we should take i18n into consideration.

**Affected Issues**

#727

### PR DESCRIPTION
**Pull request by Bowencn**
closed #806

Please make sure these boxes are checked before submitting your PR, thank you!

- [x] Make sure you follow contributing guide [English](https://github.com/element-plus/element-plus/blob/master/.github/CONTRIBUTING.en-US.md) | ([中文](https://github.com/element-plus/element-plus/blob/master/.github/CONTRIBUTING.zh-CN.md) | [Español](https://github.com/element-plus/element-plus/blob/master/.github/CONTRIBUTING.es.md) | [Français](https://github.com/element-plus/element-plus/blob/master/.github/CONTRIBUTING.fr-FR.md)).
- [x] Make sure you are merging your commits to `dev` branch.
- [x] Add some descriptions and refer to relative issues for your PR.

### CODE REVIEW

**Comment by Bowencn** [comment]
[Element Plus Playground](https://element-plus.run/#eyJBcHAudnVlIjoiPHNjcmlwdCBzZXR1cCBsYW5nPVwidHNcIj5cbmltcG9ydCB7IHJlZiwgdmVyc2lvbiBhcyB2dWVWZXJzaW9uIH0gZnJvbSAndnVlJ1xuaW1wb3J0IHsgdmVyc2lvbiBhcyBFcFZlcnNpb24gfSBmcm9tICdlbGVtZW50LXBsdXMnXG5pbXBvcnQgeyBFbGVtZW50UGx1cyB9IGZyb20gJ0BlbGVtZW50LXBsdXMvaWNvbnMtdnVlJ1xuXG5jb25zdCBwcm9wcyA9IHsgbXVsdGlwbGU6IHRydWUgfVxuXG5jb25zdCBvcHRpb25zID0gW1xuICB7XG4gICAgdmFsdWU6ICdiZWlqaW5nJyxcbiAgICBsYWJlbDogJ0JlaWppbmcnLFxuICAgIGNoaWxkcmVuOiBbXSxcbiAgICBsZWFmOiB0cnVlLFxuICB9LFxuICB7XG4gICAgdmFsdWU6ICd6aGVqaWFuZycsXG4gICAgbGFiZWw6ICdaaGVqaWFuZycsXG4gICAgY2hpbGRyZW46IFtcbiAgICAgIHtcbiAgICAgICAgdmFsdWU6ICdoYW5nemhvdScsXG4gICAgICAgIGxhYmVsOiAnSGFuZ3pob3UnLFxuICAgICAgfSxcbiAgICAgIHtcbiAgICAgICAgdmFsdWU6ICduaW5nYm8nLFxuICAgICAgICBsYWJlbDogJ05pbmdibycsXG4gICAgICB9LFxuICAgIF0sXG4gIH0sXG4gIHtcbiAgICB2YWx1ZTogJ3NoYW5naGFpJyxcbiAgICBsYWJlbDogJ1NoYW5naGFpJyxcbiAgICBjaGlsZHJlbjogW1xuICAgICAgLy8gZm9yIHRlc3Qgbm9kZXMgaW4gZGlmZmVyZW50IGxldmVscyBoYXZlIHNhbWUgdmFsdWVcbiAgICAgIHtcbiAgICAgICAgdmFsdWU6ICdzaGFuZ2hhaScsXG4gICAgICAgIGxhYmVsOiAnU2hhbmdoYWknLFxuICAgICAgfSxcbiAgICBdLFxuICB9LFxuICB7XG4gICAgdmFsdWU6ICdndWFuZ2RvbmcnLFxuICAgIGxhYmVsOiAnR3Vhbmdkb25nJyxcbiAgICBjaGlsZHJlbjogW10sXG4gIH0sXG5dXG48L3NjcmlwdD5cblxuPHRlbXBsYXRlPlxuICBcdCAgPGVsLWNhc2NhZGVyIDpvcHRpb25zPVwib3B0aW9uc1wiIDpwcm9wcz1cInByb3BzXCIgY2xlYXJhYmxlIC8+XG48L3RlbXBsYXRlPlxuIiwiUGxheWdyb3VuZE1haW4udnVlIjoiPHNjcmlwdCBzZXR1cD5cbmltcG9ydCBBcHAgZnJvbSAnLi9BcHAudnVlJ1xuaW1wb3J0IHsgc2V0dXBFbGVtZW50UGx1cyB9IGZyb20gJy4vZWxlbWVudC1wbHVzLmpzJ1xuc2V0dXBFbGVtZW50UGx1cygpXG48L3NjcmlwdD5cblxuPHRlbXBsYXRlPlxuICA8QXBwIC8+XG48L3RlbXBsYXRlPlxuIiwiaW1wb3J0X21hcC5qc29uIjoie1xuICBcImltcG9ydHNcIjoge1xuICAgIFwiZWxlbWVudC1wbHVzXCI6IFwiaHR0cHM6Ly9wcmV2aWV3LTk0OTAtZWxlbWVudC1wbHVzLnN1cmdlLnNoL2J1bmRsZS9pbmRleC5mdWxsLm1pbi5tanNcIixcbiAgICBcImVsZW1lbnQtcGx1cy9cIjogXCJ1bnN1cHBvcnRlZFwiXG4gIH1cbn0iLCJpbXBvcnQtbWFwLmpzb24iOiJ7XG4gIFwiaW1wb3J0c1wiOiB7XG4gICAgXCJ2dWVcIjogXCJodHRwczovL2Zhc3RseS5qc2RlbGl2ci5uZXQvbnBtL0B2dWUvcnVudGltZS1kb21AbGF0ZXN0L2Rpc3QvcnVudGltZS1kb20uZXNtLWJyb3dzZXIuanNcIixcbiAgICBcIkB2dWUvc2hhcmVkXCI6IFwiaHR0cHM6Ly9mYXN0bHkuanNkZWxpdnIubmV0L25wbS9AdnVlL3NoYXJlZEBsYXRlc3QvZGlzdC9zaGFyZWQuZXNtLWJ1bmRsZXIuanNcIixcbiAgICBcImVsZW1lbnQtcGx1c1wiOiBcImh0dHBzOi8vcHJldmlldy05NDkwLWVsZW1lbnQtcGx1cy5zdXJnZS5zaC9idW5kbGUvaW5kZXguZnVsbC5taW4ubWpzXCIsXG4gICAgXCJlbGVtZW50LXBsdXMvXCI6IFwidW5zdXBwb3J0ZWRcIixcbiAgICBcIkBlbGVtZW50LXBsdXMvaWNvbnMtdnVlXCI6IFwiaHR0cHM6Ly9mYXN0bHkuanNkZWxpdnIubmV0L25wbS9AZWxlbWVudC1wbHVzL2ljb25zLXZ1ZUAyL2Rpc3QvaW5kZXgubWluLmpzXCJcbiAgfSxcbiAgXCJzY29wZXNcIjoge31cbn0iLCJlbGVtZW50LXBsdXMuanMiOiJpbXBvcnQgeyBnZXRDdXJyZW50SW5zdGFuY2UgfSBmcm9tICd2dWUnXG5pbXBvcnQgRWxlbWVudFBsdXMgZnJvbSAnZWxlbWVudC1wbHVzJ1xuXG5sZXQgaW5zdGFsbGVkID0gZmFsc2VcbmF3YWl0IGxvYWRTdHlsZSgpXG5cbmV4cG9ydCBmdW5jdGlvbiBzZXR1cEVsZW1lbnRQbHVzKCkge1xuICBpZiAoaW5zdGFsbGVkKSByZXR1cm5cbiAgY29uc3QgaW5zdGFuY2UgPSBnZXRDdXJyZW50SW5zdGFuY2UoKVxuICBpbnN0YW5jZS5hcHBDb250ZXh0LmFwcC51c2UoRWxlbWVudFBsdXMpXG4gIGluc3RhbGxlZCA9IHRydWVcbn1cblxuZXhwb3J0IGZ1bmN0aW9uIGxvYWRTdHlsZSgpIHtcbiAgcmV0dXJuIG5ldyBQcm9taXNlKChyZXNvbHZlLCByZWplY3QpID0+IHtcbiAgICBjb25zdCBsaW5rID0gZG9jdW1lbnQuY3JlYXRlRWxlbWVudCgnbGluaycpXG4gICAgbGluay5yZWwgPSAnc3R5bGVzaGVldCdcbiAgICBsaW5rLmhyZWYgPSAnaHR0cHM6Ly9wcmV2aWV3LTk0OTAtZWxlbWVudC1wbHVzLnN1cmdlLnNoL2J1bmRsZS9pbmRleC5jc3MnXG4gICAgbGluay5hZGRFdmVudExpc3RlbmVyKCdsb2FkJywgcmVzb2x2ZSlcbiAgICBsaW5rLmFkZEV2ZW50TGlzdGVuZXIoJ2Vycm9yJywgcmVqZWN0KVxuICAgIGRvY3VtZW50LmJvZHkuYXBwZW5kKGxpbmspXG4gIH0pXG59IiwiX28iOnsic2hvd0hpZGRlbiI6dHJ1ZSwic3R5bGVTb3VyY2UiOiJodHRwczovL3ByZXZpZXctOTQ5MC1lbGVtZW50LXBsdXMuc3VyZ2Uuc2gvYnVuZGxlL2luZGV4LmNzcyJ9fQ==)

**Comment by Bowencn** [comment]
@sxzz hi This is my first submission and I was wondering if it takes long to check each time? Also I accidentally clicked on `update branch` Does it have any bad effects this time?

**Comment by sxzz** [comment]
💖 Thanks for your contribution! Not at all. Instead, we suggest PR is updated in time when merged.

**Comment by Bowencn** [comment]
> 💖 Thanks for your contribution! Not at all. Instead, we suggest PR is updated in time when merged.
Thanks for your reply and have a nice day!

**Comment by Bowencn** [comment]
> 💖 Thanks for your contribution! Not at all. Instead, we suggest PR is updated in time when merged.

Thanks for your reply and have a nice day!

**Comment by Bowencn** [comment]
@sxzz hi, I would like to know when the code review and merge will take place? Do I need to proactively go to the maintainer?Looking forward to your answer.

File: packages/components/cascader-panel/__tests__/cascader-panel.test.ts [hunk 1]
```diff
@@ -382,18 +382,18 @@ describe('CascaderPanel.vue', () => {
...
 
     await zjNode.trigger('click')
     const secondMenu = wrapper.findAll(MENU)[1]
-    const [hzCheckbox, nbCheckbox] = secondMenu.findAll(CHECKBOX)
+    const [, hzCheckbox, nbCheckbox] = secondMenu.findAll(CHECKBOX)
 
     await hzCheckbox.find('input').trigger('click')
     expect(hzCheckbox.classes('is-checked')).toBe(true)
     expect(zjCheckbox.classes('is-indeterminate')).toBe(true)
+    console.log(zjCheckbox)
```
Comment (stevending1st) [inline packages/components/cascader-panel/__tests__/cascader-panel.test.ts:396]: delete this line.

Comment (Bowencn) [reply packages/components/cascader-panel/__tests__/cascader-panel.test.ts]: It's been deleted.

**Pull request closed without merging**